fastai/vision/gan.py

```diff
@@ -46,7 +58,13 @@ def forward(self, x): return x.view(*(list(x.shape)+[1]*self.n_dim))
 
 # Cell
 @delegates(ConvLayer.__init__)
-def basic_generator(out_size, n_channels, in_sz=100, n_features=64, n_extra_layers=0, **kwargs):
+def basic_generator(out_size, # output size for the generator (same as the input size for the critic)
+                    n_channels, # number of channels of the output of the generator
+                    in_sz=100, # size of the input noise vector for the generator
+                    n_features=64, # number of features used in the generator
+                    n_extra_layers=0, # number of extra hidden layers in the generator
+                    **kwargs
+                    ) -> nn.Sequential:
```

Comment:
Most of the documented methods have some [style](https://github.com/muellerzr/fastai-docment-sprint/blob/master/docs_src/style-guide.md) issues. This method was missing type hints, missing the newline if not `self` or `cls`, missing capitalization, and had too many tabs for args.

If you could fix these formatting issues in other methods, everything looks pretty good.
```suggestion
def basic_generator(
    out_size:int, # Output size for the generator (same as the input size for the critic)
    n_channels:int, # Number of channels of the output of the generator
    in_sz:int=100, # Size of the input noise vector for the generator
    n_features:int=64, # Number of features used in the generator
    n_extra_layers:int=0, # Number of extra hidden layers in the generator
    **kwargs
) -> nn.Sequential:
```
One thing I am unsure of is whether we are adding docments to kwargs.

Reply:
Update. We are not adding docments to kwargs, relying on @delegates where applicable